src/main/kotlin/io/snyk/plugin/services/SnykApiService.kt

```diff
@@ -24,11 +26,14 @@ class SnykApiService : Disposable {
 
     val sastSettings: CliConfigSettings?
         get() {
-            return snykApiClient?.sastSettings(pluginSettings().organization)
+            return getSnykApiClient()?.sastSettings(pluginSettings().organization)
         }
 
     val userId: String?
-        get() = snykApiClient?.userId
+        get() = getSnykApiClient()?.getUserId()
+
+    fun reportFalsePositive(payload: FalsePositivePayload): Boolean =
```

Comment:
If you inject a SnykApiClient as a mock, it should be possible to test expected service behaviour without doing the actual network call.

Reply:
here I skip tests for SnykApiService as it just proxying call to `SnykApiClient.reportFalsePositive(payload)` and the latter tested in SnykApiClientTest